gunicorn/arbiter.py

```diff
@@ -535,7 +536,8 @@ def spawn_worker(self):
                 worker.tmp.close()
                 self.cfg.worker_exit(self, worker)
             except:
-                pass
+                self.log.error("Exception during worker exit:\n%s",
```

Comment:
I thought warning only because it doesn't affect gunicorn functions, but I don't think error would be bad.